ios/RNSScreen.mm

```diff
@@ -783,12 +778,16 @@ - (void)viewDidLayoutSubviews
   BOOL isDisplayedWithinUINavController =
       [self.parentViewController isKindOfClass:[RNScreensNavigationController class]];
   BOOL isPresentedAsNativeModal = self.parentViewController == nil && self.presentingViewController != nil;
+
   if ((isDisplayedWithinUINavController || isPresentedAsNativeModal) &&
       !CGRectEqualToRect(_lastViewFrame, self.view.frame)) {
```

Comment:
This condition: `!CGRectEqualToRect(_lastViewFrame, self.view.frame)` is not present in Fabric and it might destroy the behavior so it would be better to check it twice since there was a reason why it was not added there.

Reply:
See: [63e4938](https://github.com/software-mansion/react-native-screens/pull/1446/commits/63e49388c70440c1ed8d27fe0e0d19a0a3f2e1ae)